zer0-os/zOS
# feat: highlight matching characters when searching users and conversations

### What does this do?
When using the search function in the conversations panel, matching characters are highlighted in purple.

### Why are we making this change?
As per designs.

### How do I test this?
Type into the search conversation/user input. If any characters/words match the text in the results should be highlighted purple.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
